index.d.ts

```diff
@@ -256,9 +256,15 @@ declare namespace apm {
     disableSend?: boolean;
     elasticsearchCaptureBodyUrls?: Array<string>;
     environment?: string;
-    errorMessageMaxLength?: string; // DEPRECATED: use `longFieldMaxLength`.
+    /**
+     * @deprecated Use `longFieldMaxLength`
+     */
+    errorMessageMaxLength?: string;
     errorOnAbortedRequests?: boolean;
     exitSpanMinDuration?: string;
+    /**
+     * @deprecated Configuration 'filterHttpHeaders' is not used.
+     */
     filterHttpHeaders?: boolean;
```

Comment:
I know [I suggested](https://github.com/elastic/apm-agent-nodejs/issues/3332#issuecomment-1649865594) adding a deprecated note, but what do you think about just removing it from index.d.ts?  IIUC that would mean TypeScript users would get an error attempting to use `filterHttpHeaders` in an `apm.start(...)` call, which is reasonable for a major version bump.

Reply:
Removed!